gwtmockito/src/main/java/com/google/gwtmockito/impl/ThreadLocalCleaner.java

```diff
@@ -0,0 +1,57 @@
+package com.google.gwtmockito.impl;
+
+import java.lang.ref.WeakReference;
+import java.lang.reflect.Field;
+
+/**
+ * Utility classes to modify the private ThreadLocalMap attribute from the java.lang.Thread
+ */
+public final class ThreadLocalCleaner {
```

Comment:
```suggestion
final class ThreadLocalCleaner {
```

Reply:
The class is not in the same package that the code that uses it, so it won't work without the "public" keyword.
Maybe you want me to move it ?

Reply:
Ah, yeah, I didn't notice it in impl. Can you move it to the top-level package and make it package-private? The intention of impl is to hold things that need to be public since they're calling by generated code, but that users should never reference directly. If it doesn't need to be called by generated code it can just go in the top-level package with reduced visibility.